submissions/dashakim/memory-game/script.js

```diff
@@ -45,8 +45,8 @@ let firstCard, secondCard;
 const flipCard = e => {
   if (lockBoard) return;
   const target = e.target.parentElement;
+  if (target === firstCard) return;
   target.classList.add("flip");
-
   if (!hasFlippedCard) {
     //first click
     hasFlippedCard = true;
```

Comment:
Your `flipCard` function is almost done now to become a callback for a single event listener, added to a parent node. A few things, we got to do:
- By clicking the card - the `img` element is an event target now. That is ok, but not perfect since the element we should flip is a `div class="memory-card"`. And getting a `parentElement` of an event target doesn't give as 100% guarantee that we get an exact element. Let's resort to CSS "magic" and make always return as the target an element we exactly want. The magic CSS property is - [pointer-events](https://developer.mozilla.org/ru/docs/Web/CSS/pointer-events). Apply `none` value to all nested elements inside of `memory-card`.
- Rewrite function as event target now is `div class="memory-card"` and no more need to getting parent element.
- Add your event listener to `section` element.
- Since we adding an event listener to a section element - we want to prevent any action if clicked it directly (perhaps, somewhere between cards). Write a simple check to detect if `section` been clicked.

Reply:
A few words about method instability, when you expect the correct element by relying on nesting deepness. Short [video](https://youtu.be/bnFMnXGgnUc) example with adding padding to element describes one of the possible causes.

Reply:
Wow, that's really CSS magic. Thank's a lot, a learned few new stuff after your review. I know, my implementation isn't quite good enough. But at least a clear understand it, I hope)
Would you so kind to check if I'm improve my code just like you recommended?